lib/apiservers/portlayer/swagger.yml

```diff
@@ -207,6 +207,29 @@ paths:
           description: "error"
           schema:
             $ref: "#/definitions/Error"
+  /storage/volumestores/:
+    get:
+      description: "Get a list of available volume store locations"
+      operationId: VolumeStoresList
+      tags: ["storage"]
+      consumes:
+        - application/json
+        - application/octet-stream
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: "OK"
+          schema:
+            $ref: "#/definitions/VolumeStoresListResponse"
+        '404':
+          description: "Not found"
+          schema:
+            $ref: "#/definitions/Error"
+        default:
```

Comment:
I would prefer a status code here, and not a catch all. '500' should do the trick.